CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 * [BUGFIX] Metadata API: Mimir will now return an empty object when no metadata is available, matching Prometheus. #4782
 * [BUGFIX] Store-gateway: add collision detection on expanded postings and individual postings cache keys. #4770
 * [BUGFIX] Ruler: Support the `type=alert|record` query parameter for the API endpoint `<prometheus-http-prefix>/api/v1/rules`. #4302
+* [ENHANCEMENT] Alertmanager: Introduce new metrics from upstream. #4918
```

Comment:
I would add the list of metrics here, save users having to go digging for the information. You can add them as a second level of bullet points.